src/VolumeCache.ts

```diff
@@ -19,9 +19,9 @@ type MaybeCacheEntry = CacheEntry | null;
 type CacheEntry = {
   /** The data contained in this entry */
   // TODO allow more types of `TypedArray` to be stored together in the cache?
-  data: Uint8Array;
-  /** The subset of the volume covered by this entry, in pixels */
-  subregion: Box3;
+  data: ArrayBuffer;
+  /** The chunk of the volume covered by this entry, in pixels */
```

Comment:
Does the comment need to be updated since it's not using pixel indexing anymore?

Reply:
Whoops, yep!